From 56a786c0e1b9386817c87564c0c6b15a6f5f98d6 Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Wed, 16 Jul 2025 23:31:31 +0200
Subject: [PATCH] bin/symbols-check: add __(de)register_frame_info_bases to
 platform symbols

Cause of the following error on x86 (32-bit):

	src/gbm/libgbm.so.1.0.0: unknown symbol exported: __deregister_frame_info_bases
	src/gbm/libgbm.so.1.0.0: unknown symbol exported: __register_frame_info_bases
---
 bin/symbols-check.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/bin/symbols-check.py b/bin/symbols-check.py
index 841860c65ad53..94b3d281b4b18 100644
--- a/bin/symbols-check.py
+++ b/bin/symbols-check.py
@@ -43,6 +43,7 @@ PLATFORM_SYMBOLS = [
     '__cxa_uncaught_exceptions',
     '__cxa_unexpected_handler',
     '__deregister_frame_info',
+    '__deregister_frame_info_bases',
     '__dynamic_cast',
     '__emutls_get_address',
     '__end__',
@@ -51,6 +52,7 @@ PLATFORM_SYMBOLS = [
     '__odr_asan._glapi_Context',
     '__odr_asan._glapi_Dispatch',
     '__register_frame_info',
+    '__register_frame_info_bases',
     '_bss_end__',
     '_edata',
     '_end',
-- 
GitLab

